### ISSUE TITLE
Issues on TransCG experiment

### ISSUE THREAD

**Issue by swjeong-912d** [opipari/ClearPose#2]
Hello, 
First of all, thank you for providing high-quality DB.
We are experimenting with the TransCG network using the DB you provided, and the learning result is very different from GT. Therefore, I have 2 questions regarding this situation.

1. Recently, I checked that there was a bug in the BachNorm2d code on the TransCG side Github issue. I would like to ask if you have modified the part and tested it.
2. I would like to ask if you have changed any parameters or losses other than the ones presented in the paper.

Thank you in advance.

**Comment by cxt98** [comment]
Thanks for your interest in our dataset!

Yes we tested the TransCG code after fixing the batchnorm2d and downsample interpolation issues. We used both custom masked mse loss and entire mse loss, about 3 epochs on the training set/scenes with AdamW optimizer, lr=0.001 with combined_beta=0.001 for smooth loss, and found the accuracy of background pixels (non-transparent area) is not that high. The reason is probably because of the diversity of background depth (the original TransCG background is mostly a flat surface while ClearPose background is very diverse). However, the transparent area depth is kind of accurate, about 0.04-0.06m MAE or RMSE on ClearPose test set. My suggestion is to have another segmentation model on transparency to work together with depth completion models.
